README.md

```diff
@@ -41,6 +41,10 @@
   <img src="https://img.shields.io/github/contributors/SciTools/nc-time-axis?style=plastic"
        alt="contributors">
 </a>
+<a href="https://doi.org/10.5281/zenodo.6472641">
+  <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.6472641.svg"
+       alt="DOI">
+</a>
```

Comment:
```suggestion
<a href="https://doi.org/10.5281/zenodo.6472640">
  <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.6472640.svg"
       alt="DOI">
</a>
```

We should use the 'all version' reference, otherwise this will be a nightmare to keep up to date!

>Cite all versions? You can cite all versions by using the DOI [10.5281/zenodo.6472640](https://doi.org/10.5281/zenodo.6472640). This DOI represents all versions, and will always resolve to the latest one. [Read more](http://help.zenodo.org/#versioning).

I've tested and this change still produces a valid badge.

Reply:
Ah thanks for pointing this out!